DiscordChatExporter.Core/Exporting/Writers/Html/MessageGroupTemplate.cshtml

```diff
@@ -62,7 +64,11 @@
         <div class="chatlog__message">
             @{/* Left side */}
             <div class="chatlog__message-aside">
-                @if (isFirst)
+                @if (isSystemMessage)
+                {
+                    <svg class=chatlog__system-message-icon><use href="#icon-@Enum.GetName(typeof(MessageKind), message.Kind).ToLower()"></use></svg>
```

Comment:
This should probably work too

```suggestion
                    <svg class=chatlog__system-message-icon><use href="#icon-@message.Kind.ToString().ToLowerInvariant()"></use></svg>
```

Reply:
Yes this also works totally fine. I just read online that using toString() is bad if you use it too often because it is slow. I didn't notice any noticeable slowdown so I think it fine :)